tests/integration/main-test.js

```diff
@@ -0,0 +1,177 @@
+      'Assert library address saved previously');
+    assert.equal(find('[data-test="libPhone"]').textContent, 'Phone: (11) 4444-3333',
+      'Assert library phone saved previously');
+
+    //Filling the fields of seeder form to create libraries, books and authors with ember-faker
+    //Filling 2 to create 2 libraries
+    //Filling 4 to create 4 authors and a random number of books
+    await visit('admin/seeder');
+    await fillIn('[data-test="libInput"]', '2');
+    await pauseTest();
+    await click('[data-test="libButton"]')
+    await pauseTest();
+    await fillIn('[data-test="authorInput"]', '4');
+    await pauseTest();
+    await click('[data-test="authorButton"]')
+    await pauseTest();
+
+    //Asserting that the numbers of libraries and authors created are equal to the numbers filled in the form
+    //Remeber that the number of libraries is three because I created 1 before in the form
+    //Attention, here we are still in seeder page and the numbers I get from the respective number box.
+    assert.equal(find('[data-test="numberLib"]').textContent, '3',
+      'Assert total number of libraries in the corresponding number box');
+    assert.equal(find('[data-test="numberAuthor"]').textContent, '4',
+      'Assert total number of authors in the corresponding number box');
+
+    //The numbers of book created by the seeder is always random, so here I´m getting this random number of
+    //books from the respective number box to use later in an assertion
+    let numberBooks = (this.element.querySelector('[data-test="numberBook"]')).textContent
+    let numberBooksInt = parseInt(numberBooks)
+
+    //Asserting that there are 3 libraries cards because 1 I created in the form and two created with the 
+    //seeder
+    //Here the assertion works by counting the html elements with their respective data-test attributes,
+    //so, there must be three .card-body that means three libraries
+    await visit('libraries');
+    assert.dom('.card-body').exists({count: 3}, 'Assert the correct number of library cards');
+    await pauseTest();
+
+    //Asserting the number of authors and books created by the seeder in authors page
+    //(Here I use let numberBooksInt)
+    //Here the assertion works by counting the html elements with their respective data-test attributes.
+    //Its the same as was with the libraries, suppose there is 20 authors, there will be 20 data-test="book"
+    //elements
+    await visit('authors');
+    assert.dom('[data-test="author"]').exists({count: 4}, 'Assert the total number of authors in the table');
+    assert.dom('[data-test="book"]').exists({count: numberBooksInt}, 
+      'Assert the total number of books show in table');
+    await pauseTest();
+
+    //Invalidating the session
+    await invalidateSession();
+    
+    //I don´t know why but it doesn´t redirect to '/' just in the test as it is supposed to do after the 
```

Comment:
how about click in "sair" button

Reply:
@tpetrone  I tried and it does not work. The way to invalidate the session in a test must pass trough invalidateSession() because it is part of the ember-simple-aut/test-support. Clicking in the button don´t affect the test currentSession......